packages/eas-cli/src/commands/update/index.ts

```diff
@@ -456,6 +461,25 @@ export default class UpdatePublish extends EasCommand {
   }
 }
 
+function assertExpoUpdatesInstalled(projectDir: string, sdkVersion?: string): void {
+  // before sdk 44, expo-update was included in with the expo module
+  if (sdkVersion && semver.lt(sdkVersion, '44.0.0')) {
+    return;
+  }
+
+  const packageJsonPath = resolveFrom.silent(projectDir, './package.json');
+  if (packageJsonPath) {
+    const expoPackageJson = JsonFile.read(packageJsonPath, { json5: true });
+    if (expoPackageJson.dependencies && (expoPackageJson.dependencies as any)['expo-updates']) {
+      return;
+    }
+  }
+
+  throw new Error(
```

Comment:
If someone is building dev client and it does not heave it installed we ere asking whether we should install it for them https://github.com/expo/eas-cli/blob/main/packages/eas-cli/src/build/utils/devClient.ts#L101

maybe it would also make sense here? I don't have a strong opinion on that just a suggestion